pytket/extensions/aqt/backends/aqt_multi_zone.py

```diff
@@ -0,0 +1,403 @@
+class AQTMultiZoneBackend(Backend):
+    """
+    Interface to an AQT device with multiple compute zones.
+    """
+
+    _supports_shots = True
+    _supports_counts = True
+    _supports_contextual_optimisation = True
+    _persistent_handles = True
+
+    def __init__(
+        self,
+        architecture: MultiZoneArchitecture,
+        device_name: str = "multi_zone",
+        access_token: Optional[str] = None,
+        label: str = "",
+    ):
+        """
+        Construct a new AQT backend for multi zone architectures.
+
+        This backend currently only supports compilation of circuits
+         of type MultiZoneCircuit.
+        Submission of circuits to a quantum machine is not supported.
+
+        The backend is currently for experimental purposes only.
+
+        :param      device_name:  device name (suffix of URL, e.g. "sim/noise-model-1")
+        :type       device_name:  string
+        :param      access_token: AQT access token, default None
+        :type       access_token: string
+        :param      label:        label to apply to submitted jobs
+        :type       label:        string
+        """
+        super().__init__()
+        self._url = AQT_URL_PREFIX + device_name
+        self._label = label
+        config = AQTConfig.from_default_config_file()
+
+        if access_token is None:
+            access_token = config.access_token
+        if access_token is None:
+            raise AqtAuthenticationError()
+
+        self._header = {"Ocp-Apim-Subscription-Key": access_token, "SDK": "pytket"}
+        self._backend_info: Optional[BackendInfo] = None
+        self._qm: Dict[Qubit, Node] = {}
+        self._backend_info = fully_connected_backendinfo(
+            type(self).__name__,
+            device_name,
+            __extension_version__,
+            architecture.n_qubits_max,
+            _GATE_SET,
+        )
+        self._qm = {Qubit(i): node for i, node in enumerate(self._backend_info.nodes)}
+        self._MACHINE_DEBUG = True
+
+    def rebase_pass(self) -> BasePass:
+        return _aqt_rebase()
+
+    @property
+    def backend_info(self) -> Optional[BackendInfo]:
+        return self._backend_info
+
+    @classmethod
+    def available_devices(cls, **kwargs: Any) -> List[BackendInfo]:
+        """
+        See :py:meth:`pytket.backends.Backend.available_devices`.
+        Supported kwargs: none.
+        """
+        return []
+
+    @property
+    def required_predicates(self) -> List[Predicate]:
+        preds = [
+            NoClassicalControlPredicate(),
+            NoFastFeedforwardPredicate(),
+            NoMidMeasurePredicate(),
+            NoSymbolsPredicate(),
+            GateSetPredicate(_GATE_SET),
+        ]
+        if self._backend_info is not None:
+            preds.append(MaxNQubitsPredicate(self._backend_info.n_nodes))
+        return preds
+
+    def default_compilation_pass(self, optimisation_level: int = 1) -> BasePass:
+        assert optimisation_level in range(3)
```

Comment:
2 seems to use `FullPeepholeOptimise` instead of `SythesiseTket`. Thats the only difference I see. This was copied from the original backend in `aqt/backends/aqt.py`.